packages/labs/ssr/src/lib/module-loader.ts

```diff
@@ -301,10 +292,12 @@ const initializeImportMeta = (meta: {url: string}, module: vm.Module) => {
 
 const resolve = async (
   id: string,
-  opts: resolveAsync.AsyncOpts
+  path: string,
+  opts: Partial<enhancedResolve.ResolveOptions>
 ): Promise<string> => {
+  const resolver = enhancedResolve.create(opts);
```

Comment:
that's what i meant in my original post, i wasn't sure the best way to test resolution with bare specifiers so was hoping one of you could point me in the right direction.

the current tests use a real subdir which contains some fixture files. but how do we do the same with dependencies, without pulling the dependencies in? (or with a package.json at all)

Reply:
I think you could test that [the new `isServer` export from lit](https://github.com/lit/lit/pull/3318) is correct, since that relies on package exports working and is already there in `lit` so won't need a new dependency.

Reply:
i've added some tests to the existing module loader tests, let me know if there's any others worth adding